### PR TITLE
use alpine in fi eventing dockerfile

### PR DESCRIPTION
**Pull request by FriedrichWilken**
this PR switches from debian to alpine in `kyma/tests/fast-integration/eventing-test/image/Dockerfile` and adapts the commands for installing additional packages accordingly.

### CODE REVIEW

File: tests/fast-integration/image/Dockerfile [hunk 1]
```diff
@@ -7,18 +7,12 @@ ENV FIT_MAKE_TARGET=ci-skr
...
-    curl \
+RUN apk --no-cache add \
+    build-base \
     git \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN curl -fLSs https://deb.nodesource.com/setup_14.x | bash -
-RUN apt-get -y --no-install-recommends install nodejs
+    nodejs=16.14.2-r0
```
Comment (nachtmaar) [inline tests/fast-integration/image/Dockerfile:15]: I think it is safer to use the same nodejs version on both Prow and Spinnaker 🤔

Comment (FriedrichWilken) [reply tests/fast-integration/image/Dockerfile:15]: might makes sense. On the other hand wouldn't it be better to bump both to the latest LTS version (16.x) ?

Comment (nachtmaar) [reply tests/fast-integration/image/Dockerfile]: Also fine for me :) If it works, then I don't see a reason for not doing it :)

**Comment by nachtmaar** [comment]
We need to modify the script because it uses [bash in the shebang](e2e-fi-skr-test-dvsrd-qtj8x). Alpine comes with ash only.

**Comment by nachtmaar** [comment]
Also we need to add the npm command: `apk add npm`

**Comment by nachtmaar** [comment]
There was another problem with the image then running the skr-test:
```
ENOENT: no such file or directory, open '/KUBECONFIG/kubeconfig.yaml'
```

I did remove the env vars. They will be set by the k8s job instead.

**Review by nachtmaar — DISMISSED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Review by nachtmaar — APPROVED** [review]

**Review by jeremyharisch — APPROVED** [review]

**Pull request merged**